steering-committee/prioritization-criteria.md

```diff
@@ -0,0 +1,64 @@
+# OWD prioritization criteria
+The [OWD charter](https://github.com/openwebdocs/project/blob/main/charter.md) says _"prioritize based on the needs of the global community of web developers and designers"_. This document describes criteria for open and sponsor/vendor-neutral prioritization and will further guide OWD’s decision-making.
+
+## Criteria
+In no particular order. When deciding about what to prioritize and which opportunity to work on, OWD will take into account:
+
+### Effort
+* A higher cost ranks negatively
+* How much time needs to be spent?
+* How many pages to write?
+* How complex is the problem?
+* If new pages, how much maintenance will the pages require over time (e.g. how much in flux is the relevant technology space)?
+
+### Dependencies
+* More dependencies rank negatively
+* Does it need a change in the platform?
+* Does it need involvement from external people? (e.g. spec authors, spec tooling, etc.)
+
+### Community enablement
+* More enablement ranks positively
+* Is the work shareable?
+* Does the opportunity allow more contributors to join the project?
+* Does the opportunity allow contributors to be more effective?
+
+### Momentum
+* More momentum ranks positively
+* Are there enough implementations for a technology (and/or implementation interest)?
+* Is there a lot of community interest?
+* Has the specification stabilized? Is there no longer a risk that APIs will be renamed, redesigned etc.?
+
+### Enabling learners
+* Higher enablement ranks positively
+* Is the opportunity about concepts that need to be learned in order to understand the Web better? (as opposed to just yet another API reference documentation or such things).
+* Does the opportunity contribute an educational pathway for someone to understand a module of the mechanics of the Web?
+
+### Enabling professionals
+* Higher enablement ranks positively
+* MDN is known for being a reliable source of information for professionals. 
+* Does the opportunity address needs of professional web developers and industry experts?
+* Does the opportunity close a gap to the competition? (other platforms, native developers, proprietary tech)
```

Comment:
"to the competition" -> "between MDN and the competition" ?

Also, do you need a full stop on the end of this bullet? Your styling here is a bit inconsistent — some bullets have full stops, others do not. Choose a style and stick to it.

Reply:
I removed the full stops from all bullets. Thanks!

> "to the competition" -> "between MDN and the competition" ?

I didn't add this because I think it is meant to talk about the competition of the Web platform and not of MDN. Maybe we should make this more clear somehow? I guess the example here is that there is a super popular API or effect but it is written for say proprietary Flash Players. Web docs should show how to achieve the same effect with Web platform technologies.